client/app/components/flowchart/flowchart_directive.js

```diff
@@ -82,7 +82,7 @@ angular.module('flowChart', ['dragging'] )
 // it is painful to unit test a directive without instantiating the DOM
 // (which is possible, just not ideal).
 //
-.controller('FlowChartController', ['$scope', 'dragging', '$element', function FlowChartController($scope, dragging, $element) {
+.controller('FlowChartController', ['$scope', 'dragging', '$element', '$document', function FlowChartController($scope, dragging, $element, $document) {
```

Comment:
Umm..$scope in FlowChartController is still being used.  I haven't yet attempted to clean up flowchart_directive.js.  It still has a /* eslint-disable */ at the top.